code/modules/integrated_electronics_arfs/core/assemblies/implant.dm

```diff
@@ -0,0 +1,24 @@
+// Note that this is contained inside an actual implant subtype.
```

Comment:
Comment.

Reply:
*except it doesn't work because the UI doesn't update*

Reply:
oh

Reply:
See above about `arfs removal`